lib/scala/runtime-version-manager-test/src/test/scala/org/enso/runtimeversionmanager/config/GlobalConfigurationManagerSpec.scala

```diff
@@ -29,13 +29,14 @@ class GlobalConfigurationManagerSpec
       val configurationManager = makeConfigManager()
       val value                = Json.fromInt(42)
       configurationManager.updateConfigRaw("unknown-key", value)
-      configurationManager.getConfig.original
-        .apply("unknown-key")
-        .value shouldEqual value
-      configurationManager.removeFromConfig("unknown-key")
-      configurationManager.getConfig.original.apply(
-        "unknown-key"
-      ) should not be defined
+      configurationManager.getConfig
+        .findByKey("unknown-key") should not be defined
+      val newEmail = Json.fromString("foo@bar.com")
+      configurationManager.getConfig
+        .findByKey("author.email") should not be defined
+      configurationManager.updateConfigRaw("author.email", newEmail)
+      configurationManager.getConfig
+        .findByKey("author.email") shouldEqual newEmail.asString
```

Comment:
You have changed the behaviour of the test but kept the old test name - please update the test name to reflect the changed behaviour - otherwise this will create confusion because the test is testing something else than it says.